README.md

```diff
@@ -3,12 +3,37 @@
 * Cron Scheduler Microservice.
 
 ## Setup
+With [golang](https://golang.org/doc/install) installed
+
+Install the dependencies listed in `glide.yml` with [glide](https://github.com/Masterminds/glide)
+```
+glide install
+```
 
 Env vars
 ```bash
-export CRON_SRV_DB="host=localhost dbname=cron_srv_dev sslmode=disable"
+export CRON_SRV_DB="postgresql://postgres@localhost/cron_srv_dev?sslmode=disable"
 export CRON_SRV_PORT=3000
 ```
+> **Note:** You must have created the database 'cron_srv_dev' in postgres running at localhost (or replace with valid database name and IP);
+
+## Using
```

Comment:
Maybe we could use [this template](https://github.com/rafaeljesus/event_track#installation) instead of this one `Using` 

Followed by sections:
- Installation
- Running server
- API Documentation

wdyt?